### DISCUSSION TITLE
How to manage block governance

### DISCUSSION THREAD

**Discussion by annezazu** [WordPress/developer-blog-content#108] [Topic Ideas]
Pulling out of this issue: https://github.com/WordPress/gutenberg/issues/49592#top I think the topic of governance cuts across many audiences from higher ed to enterprise to agencies. Knowing what methods are reliable and how granular you can get is incredibly important. I think having a post outlining tried and true methods, along with perhaps a call for feedback, would help more folks save time in approaching adopting blocks in a way that works for them/their use case. This might also help us cycle feedback into the project so we can continue to improve the developer experience of managing these various pieces.

**Comment by justintadlock** [comment]
Note that there was some confusion over the term "governance" in the editorial group meeting: https://wordpress.slack.com/archives/C03RL47B3S8/p1683208076888709

It's my understanding that we're primarily referring to controlling when a block is available. Some examples might include disabling blocks altogether or contextually making blocks available to end-users (_other examples?_).

**Reply by annezazu** [reply]
I'd agree there. This could also include general management of permissions.

**Comment by ndiego** [comment]
I can pick this one up. It ties nicely with the other articles I am working on/have written.

**Comment by bph** [comment]
Conversation locked. Please continue discussion on the issue 
- #126 125

**Comment by bph** [comment]
- The issue was closed in favor of #148 
- Discussion thread closed in favor of #139
